src/state/accountCalculators.ts

```diff
@@ -60,13 +58,8 @@ export const calculateIsAccountViewOnly = createSelector(
 );
 
 /**
- * @description calculate whether the subaccount has open positions
+ * @description calculate whether the subaccount has uncommitted positions
  */
-export const calculateHasOpenPositions = createSelector(
```

Comment:
initially was updating this to use the new `num` selector, but realized it wasn't used in the codebase, so cleaned it up instead